### ISSUE TITLE
Feature: flesh out other histogram endpoints

### ISSUE THREAD

**Issue by jpmcb** [open-sauced/api#505]
### Type of feature

🍕 Feature

### Current behavior

Followup to: https://github.com/open-sauced/api/pull/501

---

The pull request histogram endpoint has been well fleshed out (since it's one of the main interfaces for data in the app and a main integration point for the v2 API migration).

The other histogram endpoints didn't get this same treatment.

### Suggested solution

Similar to `/v2/histogram/pull-requests`, we should go through the other histogram endpoints that use the base histogram DTO and allow for filtering on `repoIds`, `topics`, contributor, etc.

Note: this isn't necessary for the v2 API migration since these endpoints aren't really used yet.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

**Comment by Sadaf-A** [comment]
.take

**Comment by nickytonline** [comment]
Correct me if I'm wrong @jpmcb, but I imagine this is core team work?

**Comment by jpmcb** [comment]
Correct: this is probably something I'll tackle. @Sadaf-A - thanks for jumping in on this! Look for other issues labeled with "Good first issue" for something to get you started
